tests/test.py

```diff
@@ -0,0 +1,37 @@
+import os
+import pyChessBoardDetector
+
+def test_binding():
+    if(pyChessBoardDetector.test_bindings()== "Hello Binders!"):
+        assert pyChessBoardDetector.test_bindings() == "Hello Binders!"
+        print("Sucess")
```

Comment:
the if condition and assert here are redundant.
Read about assert
Also put more meaninful messages in the print statements
you can just do

```
def test_binding():
   assert pyChessBoardDetector.test_bindings() == "Hello Binders!", "Binding test failed"
   print("Binding test sucessful")
```